packages/block-editor/src/components/block-popover/index.js

```diff
@@ -74,12 +77,38 @@ function BlockPopover(
 		};
 	}, [ selectedElement ] );
 
+	const { isZoomOut, parentSectionBlock, isSectionSelected } = useSelect(
+		( select ) => {
+			const {
+				isZoomOut: isZoomOutSelector,
+				getSectionRootClientId,
+				getParentSectionBlock,
```

Comment:
I think we're missing whatever PR included `getParentSectionBlock`.

```
index.js:92 Uncaught TypeError: getParentSectionBlock is not a function
    at index.js:92:6
    at index.js:154:11
    at __unstableMarkListeningStores (registry.js:123:20)
    at Object.__unstableMarkListeningStores (registry.js:204:30)
    at updateValue (index.js:153:31)
    at index.js:203:3
    at useMappingSelect (index.js:228:34)
    at useSelect (index.js:317:5)
    at BlockPopover (index.js:80:72)
    at renderWithHooks (react-dom.js?ver=18:15496:20)
```

Reply:
ups! on it